vm/src/obj/objtype.rs

```diff
@@ -321,7 +321,12 @@ fn type_new_slot(metatype: PyClassRef, args: PyFuncArgs, vm: &VirtualMachine) ->
         bases
     };
 
-    let attributes = dict.to_attributes();
+    let mut attributes = dict.to_attributes();
+
+    // insert __doc__ as None if it is not included in attributes
+    if !attributes.contains_key("__doc__") {
```

Comment:
```python
>>>>> def f():
.....  pass
..... 
>>>>> f.__doc__
'The most base type'
```
The current code does not check the code above.
The expected result is shown below.
```python
>>> def f():
...  pass
... 
>>> f.__doc__
>>> 
```

Reply:
yes, this doesn't fixes the same problem in function.
I'm planning to handle that in another PR.

>This only fixes class definition.